app-admin/terraform/terraform-0.12.14.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit golang-base go-module
+
+DESCRIPTION="A tool for building, changing, and combining infrastructure safely"
+HOMEPAGE="https://www.terraform.io/"
+
+EGO_PN="github.com/hashicorp/${PN}"
+SRC_URI="https://github.com/hashicorp/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0 BSD-2 BSD-4 ECL-2.0 imagemagick ISC JSON MIT MIT-with-advertising MPL-2.0 unicode"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+DEPEND=""
```

Comment:
These empty variables can be dropped, but I don't mind about leaving them. Just FYI if you prefer it cleaner.

Reply:
I think you're right here, while I like leaving the empty variables in to remind myself for checking for new optional and required dependencies, I have to remind myself this is a go app and not C. I'll go ahead and remove them.